deploy.sh

```
@@ -0,0 +1,179 @@
+
+# Ensure SSL files exist or generate them
+if [ ! -f /etc/letsencrypt/options-ssl-nginx.conf ]; then
+  sudo wget https://raw.githubusercontent.com/certbot/certbot/main/certbot-nginx/certbot_nginx/_internal/tls_configs/options-ssl-nginx.conf -P /etc/letsencrypt/
+fi
+
+if [ ! -f /etc/letsencrypt/ssl-dhparams.pem ]; then
+  sudo openssl dhparam -out /etc/letsencrypt/ssl-dhparams.pem 2048
+fi
+
+# Create Nginx config with reverse proxy, SSL support, rate limiting, and streaming support
+sudo cat > /etc/nginx/sites-available/myapp <<EOL
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)